EMX Digital import reference error

## Type of issue
Bug @EMXDigital 

## Description
The EMX Digital adapter has reverted to importing files by absolute paths, not relative paths.  They had fixed the issue previously: https://github.com/prebid/Prebid.js/pull/3731#discussion_r274086132

## Steps to reproduce
Try importing the EMX Digital adapter.
```javascript
import prebid from 'prebid.js'
import 'prebid.js/modules/emx_digitalBidAdapter'
```

That results in this error message: 
```
ERROR in ./node_modules/prebid.js/modules/emx_digitalBidAdapter.js
Module not found: Error: Can't resolve 'src/utils' in '/Users/user/dev/video-player/node_modules/prebid.js/modules'
```


## Other information
The regression happened in this PR: https://github.com/prebid/Prebid.js/pull/3840

--- a/modules/advenueBidAdapter.js
+++ b/modules/advenueBidAdapter.js
@@ -1,6 +1,6 @@
-import { registerBidder } from 'src/adapters/bidderFactory';
-import { BANNER, NATIVE, VIDEO } from 'src/mediaTypes';
-import * as utils from 'src/utils';
+import { registerBidder } from '../src/adapters/bidderFactory';
+import { BANNER, NATIVE, VIDEO } from '../src/mediaTypes';
+import * as utils from '../src/utils';
 
 const BIDDER_CODE = 'advenue';
 const URL_MULTI = '//ssp.advenuemedia.co.uk/?c=o&m=multi';

--- a/modules/bidphysicsBidAdapter.js
+++ b/modules/bidphysicsBidAdapter.js
@@ -1,4 +1,4 @@
-import {registerBidder} from 'src/adapters/bidderFactory';
+import {registerBidder} from '../src/adapters/bidderFactory';
 import * as utils from '../src/utils';
 import {BANNER} from '../src/mediaTypes';
 

--- a/modules/cedatoBidAdapter.js
+++ b/modules/cedatoBidAdapter.js
@@ -1,6 +1,6 @@
-import * as utils from 'src/utils';
-import { registerBidder } from 'src/adapters/bidderFactory';
-import { BANNER } from 'src/mediaTypes';
+import * as utils from '../src/utils';
+import { registerBidder } from '../src/adapters/bidderFactory';
+import { BANNER } from '../src/mediaTypes';
 
 const BIDDER_CODE = 'cedato';
 const BID_URL = '//h.cedatoplayer.com/hb';

--- a/modules/digiTrustIdSystem.js
+++ b/modules/digiTrustIdSystem.js
@@ -11,7 +11,7 @@
 
 // import { config } from 'src/config';
 import * as utils from '../src/utils'
-import { ajax } from 'src/ajax';
+import { ajax } from '../src/ajax';
 import { attachIdSystem } from '../modules/userId';
 // import { getGlobal } from 'src/prebidGlobal';
 

--- a/modules/emx_digitalBidAdapter.js
+++ b/modules/emx_digitalBidAdapter.js
@@ -1,8 +1,8 @@
-import * as utils from 'src/utils';
-import { registerBidder } from 'src/adapters/bidderFactory';
-import { BANNER, VIDEO } from 'src/mediaTypes';
-import { config } from 'src/config';
-import { Renderer } from 'src/Renderer';
+import * as utils from '../src/utils';
+import { registerBidder } from '../src/adapters/bidderFactory';
+import { BANNER, VIDEO } from '../src/mediaTypes';
+import { config } from '../src/config';
+import { Renderer } from '../src/Renderer';
 import includes from 'core-js/library/fn/array/includes';
 
 const BIDDER_CODE = 'emx_digital';

--- a/modules/imonomyBidAdapter.js
+++ b/modules/imonomyBidAdapter.js
@@ -1,5 +1,5 @@
-import * as utils from 'src/utils';
-import { registerBidder } from 'src/adapters/bidderFactory';
+import * as utils from '../src/utils';
+import { registerBidder } from '../src/adapters/bidderFactory';
 
 const BIDDER_CODE = 'imonomy';
 const ENDPOINT = '//b.imonomy.com/openrtb/hb/00000';

--- a/modules/loopmeBidAdapter.js
+++ b/modules/loopmeBidAdapter.js
@@ -1,6 +1,6 @@
-import * as utils from 'src/utils';
-import { registerBidder } from 'src/adapters/bidderFactory';
-import { BANNER } from 'src/mediaTypes';
+import * as utils from '../src/utils';
+import { registerBidder } from '../src/adapters/bidderFactory';
+import { BANNER } from '../src/mediaTypes';
 
 const LOOPME_ENDPOINT = 'https://loopme.me/api/hb';
 

--- a/modules/mgidBidAdapter.js
+++ b/modules/mgidBidAdapter.js
@@ -1,7 +1,7 @@
-import {registerBidder} from 'src/adapters/bidderFactory';
+import {registerBidder} from '../src/adapters/bidderFactory';
 import * as utils from '../src/utils';
 import * as urlUtils from '../src/url';
-import {BANNER, NATIVE} from 'src/mediaTypes';
+import {BANNER, NATIVE} from '../src/mediaTypes';
 import {config} from '../src/config';
 const DEFAULT_CUR = 'USD';
 const BIDDER_CODE = 'mgid';

--- a/modules/microadBidAdapter.js
+++ b/modules/microadBidAdapter.js
@@ -1,5 +1,5 @@
-import { registerBidder } from 'src/adapters/bidderFactory';
-import { BANNER } from 'src/mediaTypes';
+import { registerBidder } from '../src/adapters/bidderFactory';
+import { BANNER } from '../src/mediaTypes';
 
 const BIDDER_CODE = 'microad';
 

--- a/modules/open8BidAdapter.js
+++ b/modules/open8BidAdapter.js
@@ -1,8 +1,8 @@
-import { Renderer } from 'src/Renderer';
+import { Renderer } from '../src/Renderer';
 import {ajax} from '../src/ajax';
 import * as utils from 'src/utils';
-import { registerBidder } from 'src/adapters/bidderFactory';
-import { VIDEO, BANNER } from 'src/mediaTypes';
+import { registerBidder } from '../src/adapters/bidderFactory';
+import { VIDEO, BANNER } from '../src/mediaTypes';
 
 const BIDDER_CODE = 'open8';
 const URL = '//as.vt.open8.com/v1/control/prebid';

--- a/modules/otmBidAdapter.js
+++ b/modules/otmBidAdapter.js
@@ -1,5 +1,5 @@
-import {BANNER} from 'src/mediaTypes';
-import {registerBidder} from 'src/adapters/bidderFactory';
+import {BANNER} from '../src/mediaTypes';
+import {registerBidder} from '../src/adapters/bidderFactory';
 
 export const spec = {
   code: 'otm',

--- a/modules/prebidmanagerAnalyticsAdapter.js
+++ b/modules/prebidmanagerAnalyticsAdapter.js
@@ -9,8 +9,8 @@ const DEFAULT_EVENT_URL = 'https://endpoint.prebidmanager.com/endpoint'
 const analyticsType = 'endpoint';
 const analyticsName = 'Prebid Manager Analytics: ';
 
-var utils = require('src/utils');
-var CONSTANTS = require('src/constants.json');
+var utils = require('../src/utils');
+var CONSTANTS = require('../src/constants.json');
 let ajax = ajaxBuilder(0);
 
 var _VERSION = 1;

--- a/modules/reklamstoreBidAdapter.js
+++ b/modules/reklamstoreBidAdapter.js
@@ -1,6 +1,6 @@
-import * as utils from 'src/utils';
-import { registerBidder } from 'src/adapters/bidderFactory';
-import { BANNER } from 'src/mediaTypes';
+import * as utils from '../src/utils';
+import { registerBidder } from '../src/adapters/bidderFactory';
+import { BANNER } from '../src/mediaTypes';
 
 const BIDDER_CODE = 'reklamstore';
 const ENDPOINT_URL = '//ads.rekmob.com/m/prebid';

--- a/modules/slimcutBidAdapter.js
+++ b/modules/slimcutBidAdapter.js
@@ -1,6 +1,6 @@
-import * as utils from 'src/utils';
-import { registerBidder } from 'src/adapters/bidderFactory';
-import { ajax } from 'src/ajax';
+import * as utils from '../src/utils';
+import { registerBidder } from '../src/adapters/bidderFactory';
+import { ajax } from '../src/ajax';
 
 const BIDDER_CODE = 'slimcut';
 const ENDPOINT_URL = '//sb.freeskreen.com/pbr';

--- a/modules/smartrtbBidAdapter.js
+++ b/modules/smartrtbBidAdapter.js
@@ -1,5 +1,5 @@
-import * as utils from 'src/utils';
-import {registerBidder} from 'src/adapters/bidderFactory';
+import * as utils from '../src/utils';
+import {registerBidder} from '../src/adapters/bidderFactory';
 const BIDDER_CODE = 'smartrtb';
 
 function getDomain () {

--- a/modules/sortableAnalyticsAdapter.js
+++ b/modules/sortableAnalyticsAdapter.js
@@ -1,10 +1,10 @@
-import adapter from 'src/AnalyticsAdapter';
-import CONSTANTS from 'src/constants.json';
-import adapterManager from 'src/adapterManager';
-import * as utils from 'src/utils';
-import {ajax} from 'src/ajax';
-import {getGlobal} from 'src/prebidGlobal';
-import { config } from 'src/config';
+import adapter from '../src/AnalyticsAdapter';
+import CONSTANTS from '../src/constants.json';
+import adapterManager from '../src/adapterManager';
+import * as utils from '../src/utils';
+import {ajax} from '../src/ajax';
+import {getGlobal} from '../src/prebidGlobal';
+import { config } from '../src/config';
 
 const DEFAULT_PROTOCOL = 'https';
 const DEFAULT_HOST = 'pa.deployads.com';

--- a/modules/timBidAdapter.js
+++ b/modules/timBidAdapter.js
@@ -1,7 +1,7 @@
-import * as utils from 'src/utils';
-import {registerBidder} from 'src/adapters/bidderFactory';
+import * as utils from '../src/utils';
+import {registerBidder} from '../src/adapters/bidderFactory';
 import * as bidfactory from '../src/bidfactory';
-var CONSTANTS = require('src/constants.json');
+var CONSTANTS = require('../src/constants.json');
 const BIDDER_CODE = 'tim';
 
 function parseBidRequest(bidRequest) {

--- a/modules/yieldoneBidAdapter.js
+++ b/modules/yieldoneBidAdapter.js
@@ -1,7 +1,7 @@
 import * as utils from '../src/utils';
 import {config} from '../src/config';
 import {registerBidder} from '../src/adapters/bidderFactory';
-import { Renderer } from 'src/Renderer';
+import { Renderer } from '../src/Renderer';
 import { BANNER, VIDEO } from '../src/mediaTypes';
 
 const BIDDER_CODE = 'yieldone';

